packages/frontend/src/utils/data-decoding.ts

```diff
@@ -43,7 +43,23 @@ export const decodeKPITokenData = async (
                 { type: "uint256", name: "initialSupply" },
             ],
             data
-        );
+        ) as [
```

Comment:
Isn't the type inferred here?

Reply:
Not anymore on the `viem` update. I'm thinking it must be an issue, but right now we have to do this...